app/v2/notifications/post_notifications.py

```diff
@@ -377,3 +435,88 @@ def get_reply_to_text(notification_type, form, template):
 
 def strip_keys_from_personalisation_if_send_attach(personalisation):
     return {k: v for (k, v) in personalisation.items() if not (type(v) is dict and v.get("sending_method") == "attach")}
+
+
+def check_for_csv_errors(recipient_csv, max_rows, remaining_messages):
+    nb_rows = len(recipient_csv)
+
+    if recipient_csv.has_errors:
```

Comment:
Going through possible cases in `has_errors` https://github.com/cds-snc/notification-utils/blob/59645a9aed395ffa08f28d567bdb8e73c9bcdfca/notifications_utils/recipients.py#L124-L132

Reply:
These errors will only be displayed to API users and not through the admin site, correct? Just want to understand if this needs to be translated at some point.